test/helm/test-values.yaml

```diff
@@ -16,7 +16,7 @@ keycloak:
       value: admin
     - name: KEYCLOAK_IMPORT
       value: /realm/alfresco-realm.json
-  postgres:
+  postgresql:
     persistence:
       existingClaim: ""
       storageClaas: "gp2"
```

Comment:
I think there is another typo: should it be `storageClass`?

Reply:
Holly... !!!!